connect-inject/container_init_test.go

```diff
@@ -142,6 +141,77 @@ consul-k8s connect-init -pod-name=${POD_NAME} -pod-namespace=${POD_NAMESPACE} \
 	}
 }
 
+func TestHandlerContainerInit_transparentProxy(t *testing.T) {
+	cases := map[string]struct {
+		globalEnabled     bool
+		annotationEnabled *bool
+		expectEnabled     bool
+	}{
+		"enabled globally, annotation not provided": {
+			true,
+			nil,
+			true,
+		},
+		"enabled globally, annotation is false": {
+			true,
+			pointerToBool(false),
+			false,
+		},
+		"enabled globally, annotation is true": {
+			true,
+			pointerToBool(true),
+			true,
+		},
+		"disabled globally, annotation not provided": {
+			false,
+			nil,
+			false,
+		},
+		"disabled globally, annotation is false": {
+			false,
+			pointerToBool(false),
+			false,
+		},
+		"disabled globally, annotation is true": {
+			false,
+			pointerToBool(true),
+			true,
+		},
+	}
+	for name, c := range cases {
+		t.Run(name, func(t *testing.T) {
+			h := Handler{EnableTransparentProxy: c.globalEnabled}
+			pod := minimal()
+			if c.annotationEnabled != nil {
+				pod.Annotations[annotationTransparentProxy] = strconv.FormatBool(*c.annotationEnabled)
+			}
+
+			expectedSecurityContext := &corev1.SecurityContext{
+				RunAsUser:  pointerToInt64(0),
+				RunAsGroup: pointerToInt64(0),
+				Capabilities: &corev1.Capabilities{
+					Add: []corev1.Capability{"NET_ADMIN"},
```

Comment:
minor nit:
```suggestion
					Add: []corev1.Capability{netAdminCapability},
```
It is bugging me that there isn't a const for NET_ADMIN in a k8s library for us somewhere :-(  I get that it's a posix thing not a k8s thing but its a common field for a security context!

Reply:
great catch! thank you!